chore(blog): march summary blog post (#1463)
asyncapi/website: pages/blog/march-2023.md

---
title: "Community Update: March 2023"
date: 2023-03-30T06:00:00+01:00
type: Communication
tags:
  - Project Status

> cover: img/posts/2023-summary/blog-banner-march.webp

Reviewer: ```suggestion
cover: /img/posts/2023-summary/blog-banner-march.webp
```

authors:
  - name: Thulisile Sibanda

>     photo: img/avatars/thulieblack.webp

Reviewer: ```suggestion
    photo: /img/avatars/thulieblack.webp
```

## Case Studies
We recently added a new [case studies](https://www.asyncapi.com/casestudies) section on our website. It showcases real-world production scenarios and how different individuals adopt and integrate our specifications and tools. On this note, we are glad to have our first official case study from Adeo featured on the website. You can read more about it [here](https://www.asyncapi.com/casestudies/adeogroup).

## Spec x Tooling
[Modelina](https://modelina.org/) website has been merged and if you are interested in generating data models for your payloads, do try it out.
Spec 3.0 meetings will now incorporate various discussions and will be the best place to ask about v3, how to use [non-JSON schemas](https://github.com/asyncapi/spec/pull/910) within AsyncAPI, and [documentation](https://github.com/asyncapi/website/issues/1433).

> ## Google Summer of Code

Reviewer 2: maybe we should mention AsyncAPI mentorship here? get update from @AceTheCreator that we are waiting for GSoC final selection to take AsyncAPI mentorship program forward? so people are aware we will do it

As a community, we are always looking for initiatives to drive diversity and bring more people passionate about open-source. In this regard, we applied to GSoC'23. Although our application didn't make it to the final organization selection, we are privileged to participate in GSoC'23 under Postman. 

Contributor applications open on the 20th of March and close on the 4th of April. If you are looking to participate we have some exciting [projects](https://github.com/postman-open-technologies/gsoc-2023/issues?q=is%3Aissue+is%3Aopen+label%3Afinal+label%3Aasyncapi+) and if you want to make your proposal outshine do watch this [session](https://www.youtube.com/live/ECe592525p4?feature=share) for tips.

> ## Ambassador Program

Reviewer 2: https://github.com/asyncapi/website/pull/917#pullrequestreview-1360772112 is almost merged, like in few days, so maybe we can already promote Ambassador landing page?

Reviewer 2: I mean instead of linking to AMBASSADORS_MEMBERS.json

Author: That makes sense, I'll link the [preview instead](https://deploy-preview-917--asyncapi-website.netlify.app/community/ambassadors) of the .json

---
title: "Community Update: March 2023"
date: 2023-03-30T06:00:00+01:00
type: Communication
tags:
  - Project Status
cover: /img/posts/2023-summary/blog-banner-march.webp
authors:
  - name: Thulisile Sibanda
    photo: /img/avatars/thulieblack.webp
    link: https://www.linkedin.com/in/v-thulisile-sibanda/
    byline: AsyncAPI Community Manager
excerpt: 'Community update for March'
featured: true
---

> March came with some ecstatic news for the AsyncAPI community. We welcomed [Thulisile Sibanda](https://www.linkedin.com/in/v-thulisile-sibanda/) as our new Community Manager. And to top it all, our beloved community guardian, [Lukasz Gornicki](https://www.linkedin.com/in/lukasz-gornicki-a621914/) stepped up to become the new [Executive Director](https://github.com/orgs/asyncapi/discussions/615) for AsyncAPI.

Reviewer 3: "our beloved community guardian Lukasz..."

kissing up even in blog posts! 😂😂😂

Reviewer 2: ![](https://media.giphy.com/media/3ohhwytHcusSCXXOUg/giphy-downsized.gif)

This is such an incredible milestone for the community and we can't wait to see the amazing things to follow in the next few months.

## AsyncAPI Conf on Tour 2023
One of our goals this year is to host [AACoT](https://opencollective.com/asyncapi/events/asyncapi-conference-on-tour-6b3c0aa1), a series of micro-conferences in different locations around the world. We are glad that our first location has been confirmed and we'll be in Madrid on the 18th - 19th of October 2023. [Call for speakers](https://sessionize.com/aacot-madrid) will be open soon and we are eagerly waiting to receive your submissions. For more details about other AACoT locations, keep an eye out for this [discussion](https://github.com/orgs/asyncapi/discussions/598).

## Case Studies

> We recently added a new [case studies](https://www.asyncapi.com/casestudies) section on our website. It showcases real-world production scenarios and how different individuals adopt and integrate our specifications and tools. On this note, we are glad to have our first official case study from Adeo featured on the website. You can read more about it [here](https://www.asyncapi.com/casestudies/adeogroup).

Reviewer 3: ```suggestion
We recently added a new [case studies](https://www.asyncapi.com/casestudies) section on our website. It showcases real-world production scenarios and how different individuals adopt and integrate our specifications and tools. On this note, we are glad to have our first official case study from [Adeo](https://www.adeo.com/en/) featured on the website. You can read more about it [here](https://www.asyncapi.com/casestudies/adeogroup).
```

We are introducing our partners in a case study so let's remember to include a courtesy backlink.

## Spec x Tooling

> [Modelina](https://modelina.org/) website has been merged and if you are interested in generating data models for your payloads, do try it out.

Reviewer 3: ```suggestion
The [Modelina](https://modelina.org/) website has been merged and if you are interested in generating data models for your payloads, do try it out.
```

Spec 3.0 meetings will now incorporate various discussions and will be the best place to ask about v3, how to use [non-JSON schemas](https://github.com/asyncapi/spec/pull/910) within AsyncAPI, and [documentation](https://github.com/asyncapi/website/issues/1433).

## Google Summer of Code
As a community, we are always looking for initiatives to drive diversity and bring more people passionate about open-source. In this regard, we applied to GSoC'23. Although our application didn't make it to the final organization selection, we are privileged to participate in GSoC'23 under Postman. 

> Contributor applications open on the 20th of March and close on the 4th of April. If you are looking to participate we have some exciting [projects](https://github.com/postman-open-technologies/gsoc-2023/issues?q=is%3Aissue+is%3Aopen+label%3Afinal+label%3Aasyncapi+) and if you want to make your proposal outshine do watch this [session](https://www.youtube.com/live/ECe592525p4?feature=share) for tips.

Reviewer 3: ```suggestion
Contributor applications open on the 20th of March and close on the 4th of April. If you are looking to participate we have some exciting [AsyncAPI projects](https://github.com/postman-open-technologies/gsoc-2023/issues?q=is%3Aissue+is%3Aopen+label%3Afinal+label%3Aasyncapi+) and if you want to make your proposal outshine do watch this [session](https://www.youtube.com/live/ECe592525p4?feature=share) for tips.
```

For accessibility, remember to always write a descriptive anchor text. Consider that "project" is too generic if read out by a screen reader.

## AsyncAPI Mentorship Program

> It's almost a year since we launched the AsyncAPI Mentorship Program and our first cohort produced [six amazing mentees](https://github.com/orgs/asyncapi/discussions/577) that truly impressed us. Several of them joined our technical steering commitee and became mentors for GSoC. With that being said, we are delighted to share that our second cohort will begin soon after GSoC'23 commences. Keep an eye out on our `#mentorship` channel for the official announcement.

Reviewer 3: ```suggestion
It's almost a year since we launched our mentorship program; our [first cohort of six AsyncAPI Mentorship Program mentees] graduated and (https://github.com/orgs/asyncapi/discussions/577) truly impressed us. Several of them joined our technical steering commitee and became mentors for GSoC. With that being said, we are delighted to share that our second cohort will begin soon after GSoC'23 commences. Keep an eye out on our `#mentorship` channel for the official announcement.
```

Accessibility improvement here as well. Consider how the phrase alone "six amazing mentees" doesn't communicate any context to a screen reader user reading a button alone.

## Ambassador Program

> One of our latest additions to the initiatives in the community is the Ambassador's Program. This program is dedicated to fostering collaboration between users and community members. It is a method to promote and nurture community growth. Presently we have a total of [5 ambassadors](https://www.asyncapi.com/community/ambassadors) and the best part is, anyone can join the program as long as they meet the requirements. You can read this [article](https://www.asyncapi.com/blog/asyncapi-ambassador-program) for more details.

Reviewer 3: ```suggestion
One of our latest additions to the initiatives in the community is the Ambassador's Program. This program is dedicated to fostering collaboration between users and community members. It is a method to promote and nurture community growth. Presently we have a total of [5 AsyncAPI Ambassadors](https://www.asyncapi.com/community/ambassadors) and the best part is, anyone can join the program as long as they meet the requirements. You can read this [article](https://www.asyncapi.com/blog/asyncapi-ambassador-program) for more details.
```

Accessibility improvement.

> One of our latest additions to the initiatives in the community is the Ambassador's Program. This program is dedicated to fostering collaboration between users and community members. It is a method to promote and nurture community growth. Presently we have a total of [5 ambassadors](https://deploy-preview-917--asyncapi-website.netlify.app/community/ambassadors) and the best part is, anyone can join the program as long as they meet the requirements. You can read this [article](https://www.asyncapi.com/blog/asyncapi-ambassador-program) for more details.

Reviewer 2: ```suggestion
One of our latest additions to the initiatives in the community is the Ambassador's Program. This program is dedicated to fostering collaboration between users and community members. It is a method to promote and nurture community growth. Presently we have a total of [5 ambassadors](https://www.asyncapi.com/community/ambassadors) and the best part is, anyone can join the program as long as they meet the requirements. You can read this [article](https://www.asyncapi.com/blog/asyncapi-ambassador-program) for more details.
```

new community pages are released 🚀

## TSC members

> Our [technical steering committee](https://www.asyncapi.com/community/tsc) has grown exponentially over the last few months. We now have a total of 44 members and a 12% increase in women maintainers so far. Shout out to our newest members, [Barbano Gonzalez](https://www.linkedin.com/in/barbano-gonzalez-moreno) and [Kenneth Aasan](https://www.twitter.com/kennethaasan) we are thrilled to have you.

Reviewer 3: ```suggestion
The [AsyncAPI TSC](https://www.asyncapi.com/community/tsc) has grown exponentially over the last few months. We now have a total of 44 members and a 12% increase in women maintainers so far. Shout out to our newest members, [Barbano Gonzalez](https://www.linkedin.com/in/barbano-gonzalez-moreno) and [Kenneth Aasan](https://www.twitter.com/kennethaasan) we are thrilled to have you.
```

## Coming up in April
We have some exciting things to look forward to in April, for instance:

- **Newsletter** - We are launching a monthly AsyncAPI community updates newsletter. This is where we will be keeping you in the loop of the activities happening in the community such as discussions, meetings, and updates. Do [subscribe](https://www.asyncapi.com/newsletter)!

> - **AsyncAPI Bounty Program** - We are launching a trial for the AsyncAPI bounty program. This program is a way to appreciate and support maintainers for the work they are doing to push the initiative forward. You can read all about it in this [discussion](https://github.com/orgs/asyncapi/discussions/513).

Reviewer 3: ```suggestion
- **AsyncAPI Bounty Program** - We are launching a trial for the AsyncAPI bounty program. This program is a way to appreciate and support maintainers for the work they are doing to push the initiative forward. You can read all about it in our [open discussion on community building/maintenance goals for 2023](https://github.com/orgs/asyncapi/discussions/513).
```

Accessibility

> - **Google Season of Docs** - After the success of [GSoD'22](https://github.com/orgs/asyncapi/discussions/303), we are happy to have been [accepted](https://developers.google.com/season-of-docs/docs/participants/) again this year. If you are a technical writer interested in open-source and improving AsyncAPI's documentation, please do [apply](https://forms.gle/pGoeakWBAg3Wfg399). Deadline is on the 15th of April.

Reviewer 3: ```suggestion
- **Google Season of Docs** - After the success of [GSoD'22](https://github.com/orgs/asyncapi/discussions/303), we are happy [AsyncAPI was accepted as a 2023 GSoD participant](https://developers.google.com/season-of-docs/docs/participants/) again this year. If you are a technical writer interested in open-source and improving AsyncAPI's documentation, we'd love to see you apply and fill out the [AsyncAPI GSoD technical writer application](https://forms.gle/pGoeakWBAg3Wfg399). Deadline is on the 15th of April.
```

> - **Call for Speakers for AACoT Madrid Edition** -  Call for Speakers for Madrid will be opening soon. You can expect a keynote from [Fran](https://www.linkedin.com/in/fmvilas/) about the history of AsyncAPI Initiative. Bookmark this [page](https://sessionize.com/aacot-madrid), and get ready to submit that talk!

Reviewer 3: ```suggestion
- **Call for Speakers for AACoT Madrid Edition** -  Call for Speakers for Madrid will be opening soon. You can expect a keynote from [Fran](https://www.linkedin.com/in/fmvilas/) about the history of AsyncAPI Initiative. Bookmark this [AACoT Madrid event page](https://sessionize.com/aacot-madrid), and get ready to submit that talk!
```
